lib/commands/general.js

```diff
@@ -118,4 +120,132 @@ commands.isKeyboardShown = async function () {
   return result.value;
 };
 
-export default commands;
+commands.mobilePressButton = async function mobilePressButton (opts = {}) {
+  const {name} = opts;
+  if (!name) {
+    log.errorAndThrow('Button Name is mandatory');
+  }
+
+  let code;
+  switch (name.toLowerCase()) {
+    case "enter":
+      code = 40;
+      break;
+    case "escape":
+      code = 50;
+      break;
+    case "arrowleft":
+      code = 75;
+      break;
+    case "arrowright":
+      code = 76;
+      break;
+    case "arrowup":
+      code = 77;
+      break;
+    case "arrowdown":
+      code = 80;
+      break;
+    case "select":
+      code = 85;
+      break;
+    case "mediafastforward":
+      code = 156;
+      break;
+    case "mediarewind":
+      code = 157;
+      break;
+    case "mediaplaypause":
+      code = 158;
+      break;
+    case "systemhome":
+      code = 218;
+      break;
+    case "systemback":
+      code = 219;
+      break;
+    case "gamepad0":
+      code = 220;
+      break;
+    case "gamepad1":
+      code = 221;
+      break;
+    case "gamepad2":
+      code = 222;
+      break;
+    case "gamepad3":
+      code = 223;
+      break;
+    case "gamepadleftbumper":
+      code = 224;
+      break;
+    case "gamepadrightbumper":
+      code = 225;
+      break;
+    case "gamepadlefttrigger":
+      code = 226;
+      break;
+    case "gamepadrightrigger":
+      code = 227;
+      break;
+    case "gamepadleftstick":
+      code = 228;
+      break;
+    case "gamepadrightstick":
+      code = 229;
+      break;
+    case "gamepadselect":
+      code = 230;
+      break;
+    case "gamepadstart":
+      code = 231;
+      break;
+    default:
+      throw new Error(`Unknown Button Name '${name.toLowerCase()}', try using 'mobile: pressKeyCode'`);
+  }
+
+  let commandObject = {
+    name: `PressKey`,
+    args: [`${code}`]
+  };
+  let commandJSON = JSON.stringify(commandObject);
+
+  let data = await this.executeSocketCommand(commandJSON);
+
+  let result;
+  try {
+    result = JSON.parse(data);
+  } catch (e) {
+    throw new Error("Bad response from PressButton");
+  }
+
+  return result.value;
+};
+
+commands.mobilePressKeyCode = async function mobilePressKeyCode (opts = {}) {
```

Comment:
Seems odd that you can specify a name for one but keycode for others

Reply:
Should I only keep KeyCode?